tr55/model.py

```diff
@@ -300,35 +314,34 @@ def postpass(tree):
     Remove volume units and replace them with inches.
     """
     if 'cell_count' in tree:
-        n = tree['cell_count']
-        tree['runoff'] = tree['runoff-vol'] / n
-        tree['et'] = tree['et-vol'] / n
-        tree['inf'] = tree['inf-vol'] / n
+        if tree['cell_count'] > 0:
+            n = tree['cell_count']
+            tree['runoff'] = tree['runoff-vol'] / n
+            tree['et'] = tree['et-vol'] / n
+            tree['inf'] = tree['inf-vol'] / n
+        else:
+            tree['runoff'] = 0
+            tree['et'] = 0
+            tree['inf'] = 0
         tree.pop('runoff-vol', None)
         tree.pop('et-vol', None)
         tree.pop('inf-vol', None)
+
     if 'distribution' in tree:
         for subtree in tree['distribution'].values():
             postpass(subtree)
 
 
-def simulate_modifications(census, cell_res=10, precolumbian=False, fn=None):
```

Comment:
Yes, the behaviour of the simulation at each cell/tile is controlled by the function `fn`.  Since that function is now a required argument, the `precolumbian` boolean no longer needs to be captured if that function is not passed.